sl beta docs (#3705)
fishtown-analytics/docs.getdbt.com: website/snippets/_sl-plan-info.md

> <span>During <a href="https://docs.getdbt.com/docs/dbt-versions/release-notes/July-2023/sl-revamp-beta#public-beta" target="_self">{props.cycle}</a>, the {props.product} is accessible to <a href="https://www.getdbt.com/pricing/">{props.plan}</a>  multi-tenant plans  <a href="https://docs.getdbt.com/docs/cloud/about-cloud/regions-ip-addresses" target="_self">{props.instance}</a> (cloud.getdbt.com login). It's available on dbt v1.6 or higher. dbt Cloud Developer plans and dbt Core users can use MetricFlow to define and test metrics locally, but can't dynamically query them with integrated tools.</span> 

Reviewer: ```suggestion
<span>During <a href="https://docs.getdbt.com/docs/dbt-versions/release-notes/July-2023/sl-revamp-beta#public-beta" target="_self">{props.cycle}</a>, you can access {props.product} on <a href="https://www.getdbt.com/pricing/">{props.plan}</a>  multi-tenant plans  <a href="https://docs.getdbt.com/docs/cloud/about-cloud/regions-ip-addresses" target="_self">{props.instance}</a> (cloud.getdbt.com login). It's available on dbt v1.6 or higher. dbt Cloud Developer plans and dbt Core users can use MetricFlow to define and test metrics locally, but can't dynamically query them with integrated tools.</span> 
```

Reviewer: I feel like this line is kind of lost in this paragraph, but maybe that's ok?
> dbt Cloud Developer plans and dbt Core users can use MetricFlow to define and test metrics locally, but can't dynamically query them with integrated tools.
